fideslib/oauth/scopes.py

```diff
@@ -2,23 +2,23 @@
 CONNECTION = "connection"
 CLIENT = "client"
 CREATE = "create"
-CREATE_OR_UPDATE = "create_or_update"
+CREATE_OR_UPDATE = "create-or-update"
```

Comment:
Looks like this is for consistency reasons, but fidesops does have some scopes that are snake-cased. Are you going to switch to using fideslib scopes? What if a user already has scopes saved associated with a token that are snake cased?

Reply:
The problem is fidesops and fidesctl have different scopes. Ideally what I would like to do is extend the scopes in fidesops and fidesctl with the ones in fideslib, but I don't see a clean way to do this. This means the scopes in fideslib aren't really being used except at user creation. In that case I had to use the ones from fideslib because it doesn't know about the others, hence the reason I was matching them up here. I do need to match these snake cases so I will update that.